### ISSUE TITLE
Fast Bridge to L2 Failing with Ledger and Blind Signing Disabled

### ISSUE THREAD

**Issue by devjumpgo** [omgnetwork/optimism#562]
Getting the following browser error:

`MetaMask - RPC Error: execution reverted: SafeERC20: low-level call failed`

Getting the following ledger error:

`Blind signing must be enabled in settings`.

Do you recommend to enable blind signing for this transaction, then disabling it afterwards? Is there a way to accomplish this bridge without blind signing?

Thanks.

**Comment by CAPtheorem** [comment]
Yes, you will need to enable blind signing. Please see the following ledger support instructions to see what blind signing is, and learn how to turn it on and why it's needed. https://support.ledger.com/hc/en-us/articles/4405481324433-Enable-blind-signing-in-the-Ethereum-ETH-app?docs=true
